x-pack/plugins/security_solution/public/management/pages/blocklist/view/components/blocklist_form.tsx

```diff
@@ -246,21 +287,21 @@ export const BlockListForm = memo<ArtifactFormComponentProps>(
         newValueErrors.VALUE_REQUIRED = createValidationMessage(ERRORS.VALUE_REQUIRED);
       }
 
-      // error if invalid hash
-      if (field === 'file.hash.*' && values.some((value) => !isValidHash(value))) {
-        newValueErrors.INVALID_HASH = createValidationMessage(ERRORS.INVALID_HASH);
-      }
-
-      const isInvalidPath = values.some((value) => !isPathValid({ os, field, type, value }));
-
-      // warn if invalid path
-      if (field !== 'file.hash.*' && isInvalidPath) {
-        newValueWarnings.INVALID_PATH = createValidationMessage(ERRORS.INVALID_PATH);
-      }
+      if (Array.isArray(values)) {
```

Comment:
as far as a i understood, the first two checks below work on hash and path, and they don't use the `IS` operator.

do you trust in us, that in case we add `IS` to path or hash, we will update this and won't introduce a bug? :) if not, i'd suggest to modify this to be universal, independently of the data type, e.g. by wrapping the string value into an array at the start of the function

Reply:
Makes sense, changed.

Reply:
thanks for the changes here!

there's an issue with handling an empty string - it's handled as an array with 1 element: `['']`, and therefore makes the length check pass

![ccccc](https://github.com/user-attachments/assets/8f677096-e53f-44d3-8042-a3372854168e)